auctioneerrpc/auctioneer.proto

```diff
@@ -545,6 +545,13 @@ message OrderMatchPrepare {
     within and the discovered market clearing price.
     */
     map<uint32, MatchedMarket> matched_markets = 10;
+
+    /*
+    The earliest absolute height in the chain in which the batch transaction can
+    be found within. This will be used by traders to base off their absolute
+    channel lease maturity height.
+    */
+    uint32 batch_height_hint = 11;
```

Comment:
We should use a new version tag for `auctioneerrpc` (for example `v1.0.3` in this case) in the main `go.mod` file when we change something. That way we only need to push a tag once we merge the PR and don't need yet another PR to bump the version _after_ pushing the tag. Saves a round trip and doesn't make anything invalid (I think) since we have a replace everywhere.

Reply:
Ah, good idea! Done.